typhos/cli.py

```diff
@@ -184,6 +184,8 @@ def create_devices(device_names, cfg=None, fake_devices=False):
 
 def typhos_run(device_names, cfg=None, fake_devices=False):
     """Run the central typhos part of typhos."""
+    typhos.utils.patch_connect_slots()
```

Comment:
Could this call be moved into the Typhos import?
I ask this because the Qt Designer plugin is also affected by this issue and adding there would probably be a centralized place to address both cases.